client/command/filesystem/memfiles-rm.go

```diff
@@ -43,7 +43,9 @@ func MemfilesRmCmd(cmd *cobra.Command, con *console.SliverConsoleClient, args []
 	}
 
 	fdInt, err := strconv.ParseInt(fdArg, 0, 64)
-
+	if err != nil {
+		con.PrintErrorf("Failed to parse fdArg: %s\n", err)
```

Comment:
We should probably return here otherwise we'll trigger other errors down the line.